### ISSUE TITLE
Enhance Why in README

### ISSUE THREAD

**Issue by lc3t35** [puzza007/katipo#5]
Can you please describe which problems you found with existing http clients, and why you cannot solve them so you wrote a new solution based on libcurl ?

**Comment by puzza007** [comment]
@lc3t35 the main reason we chose libcurl is that every protocol bug we found in existing Erlang HTTP clients (we used *4* in our project) didn't exist in libcurl.  That's the compatibility the readme talks about.  Check the number of commits https://github.com/bagder/curl has - 19766!  That's a *lot* of eyes.  Does that help?

**Comment by lc3t35** [comment]
Sorry imho it does not help, please would you mind sharing/listing the bugs you found in existing clients (which bug affects which clients) ? In your case you seem to know very well libcurl, perfect but maybe the community is not and would prefer to fix bugs in existing clients, did you submit an issue in each client repo ? It might be also interesting to let users of these clients know the limitations (or bugs) so they can choose another client or katipo ;)

**Comment by puzza007** [comment]
Sure we submitted both issues and PRs for problems we came across.  I don't think it's worth putting them in our readme however.

The community *is* fixing bugs and improving existing clients.  The line Katipo takes is that they're not doing so at the rate libcurl is.

As for client limitations, these will depend highly on how they're being used.  I really couldn't make any blanket statement on which to use.
